app.yaml

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#application: beacon-go
-#version: 1
 runtime: go
 api_version: go1
 
 handlers:
 - url: /.*
   script: _go_app
+
+env_variables:
+  GOOGLE_CLOUD_PROJECT: project-id
+  TABLE: genomics-public-data.platinum_genomes.variants
```

Comment:
If the `GOOGLE_CLOUD_PROJECT` is blank then the same logic should go for the TABLE.  Actually `TABLE` is quite vague and it should infer from the project the Beacon tables available to that project.